tools/bench/benchmark.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+
+# Benchmark runtime and memory usage of tripsu
+# Compares the working directory version against a baseline branch (main by default)
+
+set -euo pipefail
+
+### Final output path
+OUTPUT="profiling.md"
+PROFILE='release'
+BUILD_ARGS=( )
+[[ "${PROFILE}" == 'release' ]] && BUILD_ARGS+=( '--release' )
+### setup binaries
+
+# baseline binary
+BASE_BRANCH='main'
+
+BASE_DIR=$(mktemp -d)
+BASE_URL="$(git config --get remote.origin.url)"
+(
+    git clone \
+        --branch ${BASE_BRANCH} \
+        ${BASE_URL} \
+        ${BASE_DIR} \
+    && cd ${BASE_DIR} \
```

Comment:
comment: Cool, I like that you added the benchmark script!
suggestion: always quote arguments.